cluster-autoscaler/cloudprovider/azure/azure_manager.go

```diff
@@ -128,6 +130,7 @@ func createAzureManagerInternal(configReader io.Reader, discoveryOpts cloudprovi
 		Cap:      10 * time.Minute,
 	}
 
```

Comment:
We have more logic here:
```
	if !manager.azureCache.HasVMSKUs() {
		klog.Warning("No VM SKU info loaded, using only static SKU list")
		cfg.EnableDynamicInstanceList = false
	}

```

I've removed this in favor of adding it to line 68 of `azure_instance.go`. This results in us logging a new message for each call to `GetVMSSTypeDynamically`. Another option is I add this section back to `azure_manager.go` and remove the log line from the function in `azure_instance.go`. 

I think this is preferred since we already log the error returned by `getVMSSTypeDynamically` before using the static list.

Reply:
Please see a comment on `GetVMSSTypeDynamically`